### ISSUE TITLE
Adjust naming convention to reflect what's in VBMS

### ISSUE THREAD

**Issue by KHarshawat** [department-of-veterans-affairs/caseflow-efolder#168]
Today VBMS files are download with the following format

`[Claim #]-[Document Type]-[Receipt Date]-[Document ID]`

We should be consistent with this naming convention but including our numbering scheme at the bottom

`[00010]-[Claim #]-[Document Type]-[Receipt Date]-[Document ID]`

**Comment by KHarshawat** [comment]
This was based on feedback from our first eFolder Express Production Usability Session

**Comment by KHarshawat** [comment]
Just follow whatever they use in VBMS.

**Comment by lakohl** [comment]
Next step: Side by side comparison of both fetched and downloaded EX files to downloaded VBMS files in the database and VBMS files that have been downloaded.

**Comment by shanear** [comment]
waiting on guidance from @lakohl before this is development ready

**Comment by lakohl** [comment]
@orischwartz  @KHarshawat lets sync up on how to do this. Currently eX is not including the vet ID, or the received date in the filename. Trouble is, the name can quickly get so long it's very hard to read.

For now, I would recommend our current numbering system_document type_received date. I'd prefer not to have the doc ID in the filename, but if we could tag the file with the doc ID so that is visible say in Acrobat, that would be a benefit to attorneys. 

The folders are already labeled with the Vet ID and attorneys work with all the files merged into a single .pdf. However if we decide to allow single file download it will become more important to add the Vet ID in the filename.

**Comment by lakohl** [comment]
https://github.com/department-of-veterans-affairs/caseflow-efolder/issues/237 is a related issue

**Comment by lakohl** [comment]
I just realized we may not be able to avoid doc ID because it is probable that there could be more that one file with the same doc type on the same date. Our prefix number distinguishes them somewhat, but I'm wondering if this is enough.

**Comment by shanear** [comment]
@lakohl The first number will guarantee uniqueness in file name. 

The Doc ID is really long (looks like this `#{w1j3hk-j1k2l3-kjalsj3-alksjel3}`), we should not be using it in the file name.

**Comment by lakohl** [comment]
@shanear I agree re: length of Doc ID. But VBMS does use it in their file name

**Comment by lakohl** [comment]
Closing in favor of #237
